config.yml

```diff
@@ -625,92 +625,89 @@ benchmarking:
-        other_lib_name: sklearnex_LinearRegression
-
-      - title: Ridge
-        base_lib_name: sklearn_Ridge
-        other_lib_name: sklearnex_Ridge
-
-      - title: TSNE
-        base_lib_name: sklearn_TSNE
-        other_lib_name: sklearnex_TSNE
-
-      - title: PCA
-        base_lib_name: sklearn_PCA
-        other_lib_name: sklearnex_PCA
-
-  onnx:
-    n_cols: 2
-    relevant_functions:
-      - predict
-    estimators:
-      - title: KNeighborsClassifier (brute force)
-        base_lib_name: sklearn_KNeighborsClassifier_brute_force
-        other_lib_name: onnx_sklearn_KNeighborsClassifier_brute_force
-        split_bars_by_parameters:
-          - n_jobs
-
-      # - title: KNeighborsClassifier (KD tree)
-      #   base_lib_name: sklearn_KNeighborsClassifier_kd_tree
-      #   other_lib_name: onnx_sklearn_KNeighborsClassifier_kd_tree
-      #   split_bars_by_parameters:
-      #     - n_jobs
-
-      - title: HistGradientBoostingClassifier
-        base_lib_name: sklearn_HistGradientBoostingClassifier_best
-        other_lib_name: onnx_sklearn_HistGradientBoostingClassifier_best
-
-hpo_reporting:
-  estimators:
-    - name: sklearn_HistGradientBoostingClassifier
-      lib: sklearn
-      legend: scikit-learn
-      color: blue
+reporting:
+  index_notebook: index
+  reports:
+    - strategy: hp_match
+      notebook: scikit_learn_intelex_vs_scikit_learn
+      other_library: sklearnex
+      log_scale: True
+      parameters:
+        n_cols: 2
+      estimators:
+        - title: KNeighborsClassifier (brute force)
+          name: KNeighborsClassifier_brute_force
+          split_bars_by_parameters:
+            - n_jobs
 
-    - name: lightgbm
-      lib: lightgbm
-      color: red
+        - title: KNeighborsClassifier (KD tree)
+          name: KNeighborsClassifier_kd_tree
+          split_bars_by_parameters:
+            - n_jobs
 
-    - name: xgboost
-      lib: xgboost
-      color: green
+        - title: KMeans (tall)
+          name: KMeans_tall
 
-    - name: catboost_lossguide
-      lib: catboost
-      legend: catboost lossguide
-      color: purple
+        - title: KMeans (short)
+          name: KMeans_short
 
-    - name: catboost_symmetric
-      lib: catboost
-      legend: catboost symmetric
-      color: orange
+        - title: Logistic Regression
+          name: LogisticRegression
+
+        - title: LinearRegression
+          name: LinearRegression
+
+        - title: Ridge
+          name: Ridge
+
+        - title: TSNE
+          name: TSNE
+
+        - title: PCA
+          name: PCA
+
+    - strategy: hp_match
+      notebook: onnx_vs_scikit_learn
+      log_scale: True
+      other_library: onnx
+      parameters:
+        n_cols: 2
+      estimators:
+        - title: KNeighborsClassifier (brute force)
+          name: KNeighborsClassifier_brute_force
+          split_bars_by_parameters:
+            - n_jobs
+
+        # - title: KNeighborsClassifier (KD tree)
```

Comment:
```suggestion
        # Currently disabled because of a problem with matching parameters
        # when benchmarking against its ONNX version.
        # - title: KNeighborsClassifier (KD tree)
```